parser-typechecker/src/Unison/PatternMatchCoverage/PmLit.hs

```diff
@@ -0,0 +1,23 @@
+module Unison.PatternMatchCoverage.PmLit where
+
+import Unison.Prelude
+import Unison.Util.Pretty (Pretty, string)
+
+data PmLit
+  = Int Int64
+  | Nat Word64
+  | Boolean Bool
+  | Float Double
+  | Text Text
+  | Char Char
+  deriving stock (Show, Eq, Ord)
+
+prettyPmLit :: (IsString s) => PmLit -> Pretty s
```

Comment:
Not sure where these pretty prints show up; if they're just in debug messages or w/e it doesn't matter,
but if they show up to end-users I  don't think this is quite right since Unison displays Nat's and Char's differently than Haskell does 😄

Reply:
All of these pretty prints are only for debugging.